graphene/relay/node.py

```diff
@@ -92,9 +88,7 @@ def get_node_from_global_id(cls, info, global_id, only_type=None):
             _type, _id = cls.from_global_id(global_id)
         except Exception as e:
             raise Exception(
-                f'Unable to parse global ID "{global_id}". '
-                'Make sure it is a base64 encoded string in the format: "TypeName:id". '
-                f"Exception message: {str(e)}"
+                f'Unable to parse global ID "{global_id}". Make sure it is a base64 encoded string in the format: "TypeName:id". Exception message: {e}'
```

Comment:
Probably original intention was to keep lines short enough. BTW, there is no line wrap on the left side panel of github diff view.

I would suggest to keep original string, except, without `str()` call inside f-string.

Reply:
Fixed